tests/test_api.rs

```diff
@@ -1726,6 +1726,57 @@ fn promise_reject_callback_no_value() {
   }
 }
 
+#[test]
+fn promise_hook() {
+  extern "C" fn hook(
+    type_: v8::PromiseHookType,
+    promise: v8::Local<v8::Promise>,
+    _parent: v8::Local<v8::Value>,
+  ) {
+    let scope = &mut unsafe { v8::CallbackScope::new(promise) };
+    let scope = &mut v8::HandleScope::new(scope);
```

Comment:
This is unnecessary. V8 creates a HandleScope for us already. Passing the CallbackScope created 1 line above will work anywhere a HandleScope is needed.